solutions/swb-reference/src/SWBStack.ts

```diff
@@ -1469,9 +1469,13 @@ export class SWBStack extends Stack {
       domainPrefix: domainPrefix,
       websiteUrls: websiteUrls,
       userPoolName: userPoolName,
-      userPoolClientName: userPoolClientName,
+      webUiUserPoolClientName: `${userPoolClientName}-webUi`,
+      programmaticAccessUserPoolName: `${userPoolClientName}-iTest`,
       oidcIdentityProviders: [],
-      accessTokenValidity: Duration.minutes(60) // Extend access token expiration to 60 minutes to allow integration tests to run successfully. Once MAFoundation-310 has been implemented to allow multiple clientIds, we'll create a separate client for integration tests and the "main" client access token expiration time can be return to 15 minutes
```

Comment:
Have you double checked that SWB integ tests will all pass with this change? Since we're removing the admin auth from the web UI client, I dont think their current workflow will work.

Reply:
Good point. I'm going to rebuild whole SWB environment from scratch and retest.

Reply:
I'm not 100% how SWB app clients are being used so I opted to duplicate setup in `workbench-core` integration tests (both WebUI and programmatic access app clients are allowed).